docs/design/3_1_6_base_station.tex

```diff
@@ -1 +1,40 @@
-TODO: How many base stations? What will base station look like? How are machine learning results displayed?
\ No newline at end of file
+The base station in the system is a mobile device that connects to the on-board computing platform, it is used for:
+\begin{itemize}
+    \item monitoring the status of the system.
+    \item verifying the ML result.
+    \item storing the log data and video for further analysis.
+    \item control the computing platform, i.e. start object detection.
+\end{itemize}
+
+\textbf{Options for the Base Station}
+
+First option is a laptop. A laptop is a portable personal computer which has the following features:
+\begin{itemize}
+    \item It has a screen.
+    \item It has a Wi-Fi transceiver. (\textbf{F.CM.3})
+    \item It has a flash storage of at least 128GB.
+    \item It is running Windows or macOS or Linux operating system, all of them are capable of running most video processing programs.
+    \item It has an adequate processor for video processing.
+    \item It is portable and has at least 3-hour battery life. (\textbf{F.BS.3})
+\end{itemize}
+
+The second option is a tablet. Nowadays tablets are very similar to laptops in many ways. It has all the features listed above except it is running Android or iOS operating system. These mobile operating systems do not have support for running video processing programs. So it can not be used to have further analysis on the video data.
+
+\textbf{Base Station Setup}
+
+We chose the laptop as the primary base station. The primary base station is capable of doing all tasks of the base station. There could also be multiple secondary base stations. They can be either laptop or tablet. These secondary base stations may only be used to monitoring the status of the system and send control commands to the computing platform. The client can choose to use either of these base stations while testing but any further analysis on the data need to be done on the primary base station.
```

Comment:
How would we synchronize commands across base-stations? 

Perhaps instead say that, if time permits, we'll extend limited functionality to viewing results on tablets.